ibexa/experience/5.0/config/packages/security.yaml

```diff
@@ -111,16 +111,13 @@ security:
         #ibexa_oauth2_front:
         #    pattern: ^/
         #    user_checker: Ibexa\Core\MVC\Symfony\Security\UserChecker
-        #    anonymous: ~
-        #    ibexa_rest_session: ~
-        #    guard:
-        #        authenticators:
-        #            - Ibexa\Bundle\OAuth2Client\Security\Authenticator\OAuth2Authenticator
-        #            - Ibexa\PageBuilder\Security\EditorialMode\TokenAuthenticator
-        #        entry_point: Ibexa\Bundle\OAuth2Client\Security\Authenticator\OAuth2Authenticator
+        #    custom_authenticators:
+        #        - Ibexa\Bundle\OAuth2Client\Security\Authenticator\OAuth2Authenticator
+        #    entry_point: Ibexa\Bundle\OAuth2Client\Security\Authenticator\OAuth2Authenticator
```

Comment:
If the entry point is `OAuth2Authenticator`, does it even need to be set in `custom_authenticators`? Isn't it the "primary" authenticator for a firewall?

Reply:
It does - otherwise we fallback to `form_login` authenticator and oauth2 part is ignored.